tools/repl/swift/CMakeLists.txt

```diff
@@ -7,27 +7,27 @@ set(output_dir "${CMAKE_BINARY_DIR}/${CMAKE_CFG_INTDIR}/bin")
 set(module_cache_dir "${CMAKE_BINARY_DIR}/${CMAKE_CFG_INTDIR}/repl_swift_module_cache")
 
 if(NOT LLDB_BUILT_STANDALONE)
-  set(swift_depends swift)
+  set(swift_depends swift swift-stdlib)
 endif()
 
 if (CMAKE_SYSTEM_NAME MATCHES "Linux")
-  add_custom_command_target(
-      repl_swift_target
-      COMMAND "${LLDB_PATH_TO_SWIFT_BUILD}/bin/swiftc" -o "${output_dir}/repl_swift" -module-cache-path "${module_cache_dir}" -Xlinker -rpath -Xlinker \$ORIGIN/../lib/swift/linux "${CMAKE_CURRENT_SOURCE_DIR}/main.swift"
-      OUTPUT "${output_dir}/repl_swift"
-      VERBATIM
-      ALL
-      DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/main.swift" ${swift_depends})
-else()
-  add_custom_command_target(
-      repl_swift_target
-      COMMAND "${LLDB_PATH_TO_SWIFT_BUILD}/bin/swiftc" -o "${output_dir}/repl_swift" -module-cache-path "${module_cache_dir}" "${CMAKE_CURRENT_SOURCE_DIR}/main.swift"
-      OUTPUT "${output_dir}/repl_swift"
-      VERBATIM
-      ALL
-      DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/main.swift" ${swift_depends})
+  set(linux_only_flags -Xlinker -rpath -Xlinker \$ORIGIN/../lib/swift/linux)
```

Comment:
Hey Chris, can we use the RPATH Make variable here?  (Similar to a change you made up in LLVM.org LLDB?)